templates/filter_info_minimac.py

```diff
@@ -43,12 +43,13 @@ def filter_info(infoFiles, datasets, infoCutoff, out_prefix):
                     outSNP_accuracy_out.writelines(' '.join(["GROUPS"] + data) + '\\n')
             else:
                 # print info_idx, data
-                if float(data[info_idx]) >= float(infoCutoff):
+                if data[info_idx] != '-' and float(data[info_idx]) >= float(infoCutoff):
```

Comment:
This is not ok. We need to be sure that we are comparing float numbers.
We need to update this.